packages/kolibri-tools/lib/cli.js

```diff
@@ -0,0 +1,297 @@
+    <ul>`;
+    bundleData.forEach((bundle, i) => {
+      response += `<li><a href="http://${host}:${rootPort + i + 1}">${bundle.name}</a></li>`;
+    });
+    response += '</ul></body></html>';
+
+    app.use('/', (req, res) => {
+      res.send(response);
+    });
+    const server = http.createServer(app);
+    server.listen(rootPort, host, () => {
+      const url = `http://${host}:${server.address().port}`;
+      logger.info(
+        `Webpack Bundle Analyzer Reports are available at ${url}\n` + `Use ${'Ctrl+C'} to close it`
+      );
+    });
+  } else {
+    const url = `http://${host}:${rootPort + 1}`;
+    logger.info(
+      `Webpack Bundle Analyzer Report is available at ${url}\n` + `Use ${'Ctrl+C'} to close it`
+    );
+  }
+}
+
+// Build
+program
+  .command('build')
+  .description('Build frontend assets for Kolibri frontend plugins')
+  .arguments(
+    '<mode>',
+    'Mode to run in, options are: d/dev/development, p/prod/production, i/i18n/internationalization, c/clean, s/stats'
+  )
+  .option('-f , --file <file>', 'Set custom file which lists plugins that should be built')
+  .option(
+    '-p, --plugins <plugins...>',
+    'An explicit comma separated list of plugins that should be built',
+    list,
+    []
+  )
+  .option(
+    '--pluginPaths <pluginPaths...>',
+    'An explicit comma separated list of explicit file paths to plugins',
+    list,
+    []
+  )
+  .option('-m, --multi', 'Run using multiple cores to improve build speed', false)
+  .action(function(mode, options) {
+    const { fork } = require('child_process');
+    const buildLogging = logger.getLogger('Kolibri Build');
+    const modes = {
+      DEV: 'dev',
+      PROD: 'prod',
+      I18N: 'i18n',
+      CLEAN: 'clean',
+      STATS: 'stats',
+    };
+    const modeMaps = {
+      c: modes.CLEAN,
+      clean: modes.CLEAN,
+      d: modes.DEV,
+      dev: modes.DEV,
+      development: modes.DEV,
+      p: modes.PROD,
+      prod: modes.PROD,
+      production: modes.PROD,
+      i: modes.I18N,
+      i18n: modes.I18N,
+      internationalization: modes.I18N,
+      s: modes.STATS,
+      stats: modes.STATS,
+    };
+    if (typeof mode !== 'string') {
+      cliLogging.error('Build mode must be specified');
+      process.exit(1);
+    }
+    mode = modeMaps[mode];
+    if (!mode) {
+      cliLogging.error('Build mode invalid value');
+      program.help();
+      process.exit(1);
+    }
+    const multi = options.multi || process.env.KOLIBRI_BUILD_MULTI;
+    const bundleData = readWebpackJson({
+      pluginFile: options.file,
+      plugins: options.plugins,
+      pluginPaths: options.pluginPaths,
+    });
+    if (!bundleData.length) {
+      cliLogging.log('No valid bundle data was returned from the plugins specified');
+      process.exit(1);
+    }
+    const buildModule = {
+      [modes.PROD]: 'production.js',
+      [modes.DEV]: 'webpackdevserver.js',
+      [modes.I18N]: 'i18n.js',
+      [modes.STATS]: 'bundleStats.js',
+    }[mode];
```

Comment:
This currently returns `undefined` when running `make clean`

Reply:
Have brought clean in line with the other modes.